### PR TITLE
Support interactive debugger #1724

### PR DESCRIPTION
**Pull request by Bilal-S**
Feature Request Template ------------------------------------------

##### Feature Request:
Interactive Debugger for Serverless
I am expanding this from issue #1720 
The need existed for me to understand more clearly how the framework operates. An interactive debugging connection seems the solution. I am proposing the use of node-inspector which allows a connection to a browser window in which the code can be reviewed while running.

##### Benefits:

* interactive debugging
* set breakpoints, trace call history, call stack, interactive command window, etc.
* tremendous speed up for people contributing to the platform

##### Drawbacks:

* There does not seem to be slick way of exiting the debugging session. The V8 profiler window in browser will need to be closed by developer.
* pre-requisites exist: installation of node-inspector package and chrome browser

##### Additional Details:

Discussion template  ----------------------------------------------

##### TLDR:
Adding connection to node-inspector so interactive debugging can occur while running serverless commands.

### CODE REVIEW

**Comment by Bilal-S** [comment]
A need for debugging your local code also exists with local/offline development. If we could include this in future versions a quick start local dev environment would be possible.
For example with a plug-in like serverless offline:
[https://github.com/dherault/serverless-offline/issues/45](url)
